USca/USca_RTU/Processor/Simulator/SimTank.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace USca_RTU.Processor.Simulator
+{
+    public partial class SimTank : INotifyPropertyChanged
+    {
+        public int Address { get; set; }
+        public string Name { get; set; } = "";
+        private double _value;
+        public double Value { get { return _value; } set { _value = Math.Max(0, value); } }
```

Comment:
This is the only simulated object where we do value clamping, and without a max value at that. Is this deliberate?

Reply:
It doesn't make sense for a tank to have negative amount of something. It's either empty or has something in it. There's no upper boundary because then I would also have to simulate "spilling" and giving the tank a maximum capacity. It's too much work. We have alarms, that's enough.